src/main.js

```diff
@@ -952,3 +952,12 @@ class MiniGraphCard extends LitElement {
 }
 
 customElements.define('mini-graph-card', MiniGraphCard);
+
+// Configure the preview in the Lovelace card picker
+window.customCards = window.customCards || [];
+window.customCards.push({
+  type: 'mini-graph-card',
+  name: 'Mini Graph Card',
+  preview: false,
+  description: 'The Mini Graph card is a minimalistic and customizable graph card',
```

Comment:
Remove the comma.

Reply:
yes I tested successfully - removing the comma gives a 'missing trailing comma (comma-dangle)' error when running through npm

Reply:
From the last object, of course. https://eslint.org/docs/rules/comma-dangle

Reply:
Can see you've approved now, are there any changes required?

Reply:
```suggestion
  description: 'The Mini Graph card is a minimalistic and customizable graph card'
```

Reply:
You might have to bare with me as I'm new to js and npm, so sorry if I'm missing something. But like I said above, removing that comma gives a 'missing trailing comma (comma-dangle)' error when running through 'npm run build'. Comma dangle seems to be set to always-multiline (or possibly always). So my understanding is that no change is required.